VERSION

```diff
@@ -1 +1 @@
-1.31.8
\ No newline at end of file
+1.32.0
```

Comment:
Just for clarity, Expeditor will automatically bump this to 1.32.1 (unless you skip the version bump)